pkg/trace/config/testdata/ok_apm_config_env.yaml

```diff
@@ -0,0 +1,7 @@
+hostname: thing
```

Comment:
Can we put these files in a sub-folder? Something like `env` or `envtests`? Maybe not a good idea, but just throwing it out there. Alternatively we could prefix them all with `env_`

Reply:
Sub-folder works for me!

Reply:
Actually I think renaming to have `unnormalized_env_*` and `ok_env_*` works out better.